modules/social_features/social_page/social_page.install

```diff
@@ -238,7 +238,7 @@ function social_page_update_10302() {
 /**
  * Remove deprecated permission "override page revision log entry".
  */
-function social_page_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.